lib/ecto/adapters/sql/sandbox.ex

```diff
@@ -221,16 +221,16 @@ defmodule Ecto.Adapters.SQL.Sandbox do
           ** (DBConnection.ConnectionError) owner #PID<> timed out
           because it owned the connection for longer than 15000ms
 
-  If you have a long running test, the timeout for the connection ownership may
+  If you have a long running test (or you're debugging with IEx.pry), the timeout for the connection ownership may
   be too short.  You can increase the timeout by setting the
-  `:ownership_timeout` options for your repo config:
+  `:ownership_timeout` options for your repo config (preferably in `config/test.exs`):
```

Comment:
Ah, I was not aware we're already using it. So I guess it's fine. I would only change it to mention `config.exs` first, and later the `test.exs`, just like the fragment you're citing.